pkg/srv/mstream/api.go

```diff
@@ -132,6 +132,16 @@ func (s *ApiServer) configureRouter() {
 	//   "400":
 	//     "$ref": "#/responses/badReq"
 	subRouter.HandleFunc("/flush", s.handleFlush()).Methods("GET")
+	// swagger:operation GET /headway mstream getHeadway
+	// ---
+	// summary: headway mstream
+	// description: --
+	// responses:
+	//   "200":
+	//     "$ref": "#/responses/okResp"
+	//   "400":
+	//     "$ref": "#/responses/badReq"
+	subRouter.HandleFunc("/progress", s.handleProgress()).Methods("GET")
```

Comment:
I believe this api handler must per device. Smth like /progress/<device_name>

Reply:
Even more it is rather /last_event/<device_name>

Reply:
I am fixed it